pkg/provisioner/cloudprovisioner.go

```diff
@@ -60,16 +64,23 @@ type listVolumeStatus struct {
 	err           error
 }
 
-func NewCloudProvisioner(kubeClient clientset.Interface, topologyKey string) (*CloudProvisioner, error) {
+func NewCloudProvisioner(kubeConfig *rest.Config, kubeClient clientset.Interface, topologyKey string, namespace string) (*CloudProvisioner, error) {
 	azCloud, err := azureutils.GetAzureCloudProvider(kubeClient)
 	if err != nil {
 		return nil, err
 	}
 
+	diskClient, err := azureutils.GetAzDiskClient(kubeConfig)
+	if err != nil {
+		return nil, err
+	}
+
 	topologyKeyStr = topologyKey
 
 	return &CloudProvisioner{
-		cloud: azCloud,
+		azDiskClient: diskClient,
+		cloud:        azCloud,
+		namespace:    namespace,
```

Comment:
@landreasyan @sunpa93  Do we have an use case where `azDiskClient` or `namespace` is getting used?

Reply:
I forgot to revert this change. I will go ahead and revert it :)